pkg/sql/pgwire/types.go

```diff
@@ -178,7 +179,7 @@ func writeTextDatumNotNull(
 		b.writeLengthPrefixedDatum(v)
 
 	case *tree.DBytes:
-		writeTextBytes(b, string(*v), conv)
+		writeTextBytes(b, unsafeBytesToString([]byte(*v)), conv)
```

Comment:
Hmm I feel like there is something weird going on here - a `DBytes` is a string already. Calling string(*v) where v is a pointer to DBytes shouldn't cause a copy? What is the expense here exactly?

Reply:
Yeah, you're right, I just got overzealous and wasn't paying attention. Give me a sec.